lighthouse-core/report/templates/report-template.html

```diff
@@ -153,9 +153,9 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
                 <li class="subitem {{#if subItem.comingSoon}}--coming-soon{{/if}} {{#if (shouldShowHelpText subItem.score)}}--show-help{{/if}}">
 
                   <p class="subitem__desc">
-                    {{#unless ../../scored }}
+                    <!--{{#unless ../../scored }}
```

Comment:
leaving category in is fine, I'm sure #1463 will have something to say about it anyways, but I'd say just delete this. We definitely don't want.